core-aam/core-aam.html

```diff
@@ -215,23 +215,23 @@
 	<ul>
 		<li>Direct property accessors to the current values - <code>Current{PropertyName}</code>, e.g. <code>IUIAutomationElement::CurrentName</code> for the <code>Name</code> property</li>
 		<li>Cached property accessors - <code>Cached{PropertyName}</code>, e.g. <code>IUIAutomationElement::CachedName</code> for the <code>Name</code> property. Using cached values is preferred when providers and clients are used in remote environments.</li>
-		<li><code>GetCurrentPropertyValue</code> and passing UIA Property ID enumeration value corresponding to that property to get current value, e.g. <code>IUIAutomationElement::GetCurrentPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.</li>
-		<li><code>GetCachedPropertyValue</code> and passing UIA Property ID enumeration value corresponding to that property to get cached value, e.g. <code>IUIAutomationElement::GetCachedPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.
+		<li><code>GetCurrentPropertyValue</code> and passing the UIA Property ID enumeration value corresponding to that property to get the current value, e.g. <code>IUIAutomationElement::GetCurrentPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.</li>
+		<li><code>GetCachedPropertyValue</code> and passing the UIA Property ID enumeration value corresponding to that property to get the cached value, e.g. <code>IUIAutomationElement::GetCachedPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.
       	</ul>
-	<p>Properties for specific UIA control patterns are queried the same way using relevant control pattern interfaces. Taking Toggle Pattern as an example, to query ToggleState property clients can use IUIAutomationTogglePattern::CurrentToggleState or IUIAutomationTogglePattern::GetCurrentPropertyValue(UIA_ToggleToggleStatePropertyId) to get the current value.</p>
-      	<p>Note: The property mapping in this specification provide the <code>{PropertyName}</code> and don't specify all specific ways to access the property value. Automation clients can access current or cached values using conventions described above, depending on specific needs and coding style conventions.</p>
+	<p>Properties for specific UIA control patterns are queried the same way using relevant control pattern interfaces. Taking Toggle Pattern as an example, to query the ToggleState property clients can use IUIAutomationTogglePattern::CurrentToggleState or IUIAutomationTogglePattern::GetCurrentPropertyValue(UIA_ToggleToggleStatePropertyId) to get the current value.</p>
+      	<p>Note: The property mapping in this specification provide the <code>{PropertyName}</code> and do not specify all specific ways to access the property value. Automation clients can access current or cached values using conventions described above, depending on specific needs and coding style conventions.</p>
```

Comment:
Thanks @boggydigital !

One more suggested change that has to do with W3C rules.  The intro ends in:
"Note:  The property mapping in this specification ..."

Notes in W3C specs are informative only, and add no normative requirements.  Notes are styled differently as well.  For example see the note after the bulleted list in the [focus events table section](https://rawgit.com/w3c/aria/master/core-aam/core-aam.html#focus_state_event_table).

IMHO, this isn't a note, and the "Note:" text should be removed.  But, if you feel differently, and that this text is truly is a note, remove the "Note:" text _and_ add class="note" to the `<p>` element.